### DISCUSSION TITLE
Add Merge function or enhance the Override menu to include modification capabilities for routing rules. 增加Merge功能或者在Override菜单中增加对于分流规则的修改功能

### DISCUSSION THREAD

**Discussion by Aether-a** [MetaCubeX/ClashMetaForAndroid#237] [Ideas]
Currently, this app only supports overwriting custom items in the override menu and cannot achieve the higher degree of custom configuration for configuration files like Clash Verge. The Merge function is very important, especially because many predefined configuration files often have routing rules that do not suit the user. However, since the nodes included in subscriptions often need to be updated, manually modifying and re-importing configuration files is very troublesome. If the Merge function could be added, users would be able to customize any subscribed configuration within the app, greatly enhancing convenience. It is hoped that this feature can be added or that the customizable items in the override menu can be expanded.
目前，这款应用程序仅支持在覆盖菜单中覆盖自定义项目，无法实现像 Clash Verge 这样对配置文件的高级自定义配置。合并功能非常重要，尤其是因为许多预定义的配置文件中的路由规则往往不适合用户使用。然而，由于订阅中包含的节点经常需要更新，手动修改和重新导入配置文件非常麻烦。如果能够添加合并功能，用户将能够在应用程序内自定义任何订阅的配置，大大提高便利性。希望能添加此功能或扩展覆盖菜单中的可自定义项目。

**Comment by gituser056391** [comment]
同求，很需要这个功能
